redfish_guide/master.adoc

```diff
@@ -12,11 +12,11 @@
 // caller, see https://github.com/ManageIQ/manageiq_docs/tree/master/common/attributes.
 //
 
-:product-title: ManageIQ
-:product-title_short: ManageIQ
-:product-title_short_l: mangeiq
-:product-title_abbr: miq
-:product-title_abbr_uc: MIQ
+//:product-title: ManageIQ
+//:product-title_short: ManageIQ
+//:product-title_short_l: mangeiq
+//:product-title_abbr: miq
+//:product-title_abbr_uc: MIQ

```

Comment:
These lines should be commented out as per instructions in the README file. The uncommented lines got to the commit by mistake.